cluster-autoscaler/cloudprovider/aws/aws_util.go

```diff
@@ -20,17 +20,20 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/ec2metadata"
-	"github.com/aws/aws-sdk-go/aws/endpoints"
-	"github.com/aws/aws-sdk-go/aws/session"
+	"io"
 	"io/ioutil"
```

Comment:
Looks like we no longer use this import, can you please remove it as it'll cause tests to fail otherwise.

Reply:
It's used here: https://github.com/kubernetes/autoscaler/pull/4199/files#diff-17200bf96aa960fa662c306249293f2fa99282cd10040050f1107b901db43012R134

(imports here are generated by goimports)

Reply:
Ah, must have changed between rebasing. Otherwise goimports would have fixed it for me. I'll sort that now.